test/unit/unit/format-ja.js

```diff
@@ -0,0 +1,48 @@
+define([
+	"cldr",
+	"src/core",
+	"src/unit/format",
+	"src/unit/properties",
+	"json!cldr-data/main/ja/units.json",
+	"json!cldr-data/supplemental/likelySubtags.json"
+], function( Cldr, Globalize, formatUnit, unitProperties, jaUnits, likelySubtags ) {
+
+var cldr, globalize;
+
+Cldr.load( jaUnits );
+Cldr.load( likelySubtags );
+
+globalize = new Globalize( "ja" );
+cldr = globalize.cldr;
+
+QUnit.module( "Unit Format - Compound Form with no one property in cldr" );
+
+function oneOrOtherPluralGenerator() {
+	return "other";
+}
+
+function stubNumberFormatter( number ) {
+	return number.toString();
+}
+
+QUnit.assert.unitFormatJa = function ( value, unit, options, expected ) {
+	var unitProps = unitProperties( unit, options.form, cldr );
+
+	this.equal(
+		formatUnit( value, options.numberFormatter || stubNumberFormatter, oneOrOtherPluralGenerator, unitProps ),
+		expected
+	);
+};
+
+QUnit.test( "Compound form (without precomputed)", function ( assert ) {
+	assert.unitFormatJa( 1, "length-foot-per-second", { form: "long" }, "1 フィート毎秒" );
+	assert.unitFormatJa( 100, "length-foot-per-second", { form: "long" }, "100 フィート毎秒" );
+	assert.unitFormatJa( 1, "megabyte-per-second", { form: "narrow" }, "1MB/秒" );
+	assert.unitFormatJa( 100, "megabyte-per-second", { form: "narrow" }, "100MB/秒" );
+
+	assert.unitFormatJa( 1.2345678910, "megabyte-per-second",
+		{ form: "narrow", numberFormatter: function (number) { return number.toFixed(1); }},
+		"1.2MB/秒" );
+});
+
+});
```

Comment:
Please, can we include these tests in the existing [test/unit/unit/format.js](https://github.com/globalizejs/globalize/blob/master/test/unit/unit/format.js)?